### PR TITLE
Add configuration documentation for Cloud AWS SNS

### PR DESCRIPTION
**Pull request by car12o**
##### Summary

Add configuration documentation for Cloud AWS SNS.

### CODE REVIEW

**Comment by ilyam8** [comment]
Hey, @car12o. You need to update [metadata](https://github.com/netdata/netdata/blob/master/health/notifications/awssns/metadata.yaml).  The doc is generated from it.

**Comment by car12o** [comment]
> Hey, @car12o. You need to update [metadata](https://github.com/netdata/netdata/blob/master/health/notifications/awssns/metadata.yaml). The doc is generated from it.

I may be missing something but this is a cloud integration not an agent, do I need to update that file? Never have done it before :sweat_smile:

**Comment by ilyam8** [comment]
@car12o I think you are right, I was confused by https://github.com/netdata/netdata/tree/master/health/notifications/awssns#aws-sns - we have Agent AWS SNS notification method too, sorry 🙈

**Comment by Ancairon** [comment]
Hey hey, metadata 🚔 here, 😅 we do have metadata for these too!

https://github.com/netdata/netdata/blob/master/integrations/cloud-notifications/metadata.yaml

You need to edit that for metadata to appear as md, all changes are made there.


If you take a look at current MD files they have a nice capital letter message at the top saying they are auto generated.

**Comment by car12o** [comment]
Thanks guys, just updated the file.

Something that concerns me is the fact that we duplicate content on two files, it's very easy to update just one file and forget about the other :sweat_smile:

File: integrations/cloud-notifications/metadata.yaml [hunk 1]
```diff
@@ -279,6 +279,54 @@
         * **Integration configuration** are the specific notification integration required settings, which vary by notification method. For RocketChat:
           - Webhook URL - URL provided on RocketChat for the channel you want to receive your notifications.
 
+- id: 'notify-cloud-awssns'
+  meta:
+    name: 'AwsSns
```
Comment (ilyam8) [inline integrations/cloud-notifications/metadata.yaml:284]:
```suggestion
    name: AwsSns
```

**Comment by ilyam8** [comment]
@car12o update just meta, ingnore `docs/cloud/alerts-notifications/`.I think these files are not used and we need to delete them.

**Comment by ilyam8** [comment]
> and we need to delete them.

https://github.com/netdata/netdata/pull/16372

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Comment by car12o** [comment]
> @car12o update just meta, ignore `docs/cloud/alerts-notifications/`.I think these files are not used and we need to delete them.

wait if that's the case we are removing all the images that help the user to configure the integration.
we should use the the original `.md` files instead of the generated one.

**Comment by tkatsoulas** [comment]
Guys always prefer svgs over png https://deploy-preview-1881--netdata-docusaurus.netlify.app/docs/alerting/notifications/centralized-cloud-notifications/amazon-sns

**Comment by tkatsoulas** [comment]
cc @hugovalente-pm @car12o @ilyam8

**Pull request merged**